### PR TITLE
Add What's New Popup for Petnames

### PR DESCRIPTION
**Pull request by dbrans**
## **Description**
Adds a what's new popup for Petnames.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

### **After**

image

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: shared/notifications/index.js [hunk 1]
```diff
@@ -494,5 +503,19 @@ export const getTranslatedUINotifications = (
...
     ///: END:ONLY_INCLUDE_IF
+    [NOTIFICATION_PETNAMES]: {
+      ...UI_NOTIFICATIONS[NOTIFICATION_PETNAMES],
+      title: t('notificationsPetnamesTitle'),
+      description: [
+        t('notificationsPetnamesDescriptionOne'),
+        t('notificationsPetnamesDescriptionTwo'),
+      ],
+      actionText: t('notificationsPetnamesActionText'),
+      date: UI_NOTIFICATIONS[NOTIFICATION_PETNAMES].date
```
Comment (matthewwalsh0) [inline shared/notifications/index.js:514]: If the date is currently `null`, do we need this logic here now or can we also hardcode `''`?

File: shared/notifications/index.js [hunk 2]
```diff
@@ -494,5 +503,19 @@ export const getTranslatedUINotifications = (
...
+        ? new Intl.DateTimeFormat(formattedLocale).format(
+            new Date(UI_NOTIFICATIONS[NOTIFICATION_OPEN_BETA_SNAPS].date),
```
Comment (matthewwalsh0) [inline shared/notifications/index.js:516]: If it's a standard and we want this logic anyway, should this be `NOTIFICATION_PETNAMES`?

Comment (dbrans) [reply shared/notifications/index.js:516]: You caught my copy-paste error. Thank you!

File: shared/notifications/index.js (hunk 1 above)
Comment (dbrans) [reply shared/notifications/index.js:514]: Done.

**Review by matthewwalsh0 — APPROVED** [review]

**Comment by coreyjanssen** [comment]
looks good to me!

**Review by sleepytanya — APPROVED** [review]

**Pull request merged**